File: API/Models/Solicitacao.cs
// PROP - cria a propriedade
using System;
using System.Collections.Generic;

namespace API.Models
{
    public class Solicitacao

    {
        public Solicitacao() => CriadoEm = DateTime.Now;

        public int Id { get; set; }
        public string Funcionario { get; set; }
        public Pedido Pedido { get; set; }
        public int PedidoId { get; set; }
        public List<Despesa>  Despesas  { get; set; }
        public DateTime CriadoEm { get; set; }

        
        
    }
}

File: FRONT/src/app/models/solicitacao.ts
import { Pedido } from './pedido';
import { Despesa } from './despesa';


export interface Solicitacao {
    id: number;
    funcionario: String;
    Pedido?: Pedido;
    pedidoId: string;
    Despesas?: Despesa;
    criadoem?: string;
}
